source/crud/tests/README.md

```diff
@@ -737,3 +737,10 @@ that `firstEvent.operationId` is equal to `secondEvent.operationId`. Assert both
 To force completion of the `w:0` writes, execute `coll.countDocuments` and expect the returned count is
 `maxMessageSizeBytes / maxBsonObjectSize + 1`. This is intended to avoid incomplete writes interfering with other tests
 that may use this collection.
+
+### 16. Generated document identifiers are the first field in their document
+
+Construct a `MongoClient` (referred to as `client`) with
+[command monitoring](../../command-logging-and-monitoring/command-logging-and-monitoring.md) enabled to observe
+CommandStartedEvents. Perform an `insert` command using `client` and assert that one CommandStartedEvent (referred to as
```

Comment:
> Perform an `insert` command

This is ambiguous, as the prose test could be implementing using the generic command runner, in which case I'd expect no ID generation. I'd suggest changing this to explicitly suggesting running an `insertOne` operation. You can extend that with `insertMany` and a `bulkWrite` if you want full test coverage.